app/mod_auth/region_manager.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2014 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+
+# Import flask dependencies
+from flask import abort, Response
+
+from app.settings.log import logger
+from functools import wraps
+from app.mod_auth.AuthorizationManager import AuthorizationManager
+from app.settings import settings
+import requests
+import json
+import httplib
+
+__author__ = 'fla'
+
+
+class region():
+    regions = []
+    ERROR_MESSAGE = '{ "error": { "message": "Bad region", "code": 410 } }'
```

Comment:
Is this a HTTP Status code `410` (`GONE`) or a custom error code?
* In the former case, please use constants defined by `httplib`
* In the later case, please label such custom errors with the corresponding constants